packer/conf/buildkite-agent/hooks/environment

```diff
@@ -8,6 +8,9 @@ source ~/cfn-env
 echo "~~~ :llama: Setting up elastic stack environment ($BUILDKITE_STACK_VERSION)"
 cat ~/cfn-env
 
+echo "Fixing permissions for '${BUILDKITE_AGENT_NAME}'..."
+sudo -E /usr/bin/fix-buildkite-agent-builds-permissions
```

Comment:
Passing in all env means I can override PATH and things for nefearious purposes. I think the interface should be kept very minimal and require passing in the variables as arguments.